problems/strings.js

```diff
@@ -87,14 +123,40 @@ function removeLastWord() {}
  * @returns {string}
  */
 
-function everyOtherLetter() {}
+function everyOtherLetter(str) {
+  let output = "";
+  for (let i = 0; i < str.length; i++) {
```

Comment:
Looks good, but if we change step to be i += 2 things are even cleaner. No checks required